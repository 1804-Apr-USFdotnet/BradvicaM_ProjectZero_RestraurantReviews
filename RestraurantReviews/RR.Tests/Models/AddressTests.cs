using ApprovalTests;
using ApprovalTests.Reporters;
using Microsoft.VisualStudio.TestTools.UnitTesting;
using RR.Models;

namespace RR.Tests.Models
{
    [TestClass]
    public class AddressTests
    {
        [TestMethod]
        [UseReporter(typeof(DiffReporter))]
        public void ToString_OnCall_DisplaysProperly()
        {
            var address = new Address
            {
                City = "Kansas City",
                State = "MO",
                Street = "123 Barbeque St.",
                ZipCode = 81721
            };

            Approvals.Verify(address);
        }
    }
}
