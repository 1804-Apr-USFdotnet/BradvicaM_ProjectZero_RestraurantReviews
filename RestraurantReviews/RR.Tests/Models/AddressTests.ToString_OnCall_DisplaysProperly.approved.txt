123 Barbeque St. Kansas City MO 81721